### ISSUE TITLE
Package is not working on PHP 7.0

### ISSUE THREAD

**Issue by mauro-moreno** [igaster/laravel-theme#39]
`$ Type error: Argument 1 passed to igaster\laravelTheme\Tree\Item::addParent() must be an instance of igaster\laravelTheme\Tree\Item, boolean given, called in /home/autocity/website2016/releases/20160719005736/vendor/igaster/laravel-theme/src/Themes.php on line 32`

**Comment by igaster** [comment]
I've test it on PHP 7 and it works fine. It seems that there is an error in your configuration (themes.php). Maybe an orphan theme? If you can share your configuration file I might be able to help you...

**Comment by mauro-moreno** [comment]
On Themes::add

```
public function add(Theme $theme, $parentName = ''){
            $theme->addParent($parentName ? $this->find($parentName) : $this->root);
            return $theme;
}
```

`$this->find($parentName)` can return `false` and PHP 7 is typed so what I've done was:

```
public function add(Theme $theme, $parentName = '') {
    $parent = $this->find($parentName);
    $theme->addParent($parent ? $parent : $this->root);
    return $theme;
}
```

**Comment by igaster** [comment]
Shoud be fixed in v1.1.3 release
